elements/lisk-p2p/src/p2p.ts

```diff
@@ -289,10 +290,15 @@ export class P2P extends EventEmitter {
 
 		// This needs to be an arrow function so that it can be used as a listener.
 		this._handlePeerPoolRPC = (request: P2PRequest) => {
+			// Process protocol messages
 			if (request.procedure === REMOTE_EVENT_RPC_GET_PEERS_LIST) {
 				this._handleGetPeersRequest(request);
 			}
 
+			if (request.procedure === REMOTE_EVENT_RPC_GET_NODE_INFO) {
+				request.end(this._nodeInfo);
```

Comment:
I saw a convention used in code `_handle<Request>`, please use same instead of manipulating `request` object directly.